docs/patch-body-parameters-schema.md

```diff
@@ -0,0 +1,52 @@
+# PatchBodyParametersSchema
+
+## Description
+
+ A request parameter of the Patch Operation must not have a required/default/'x-ms-mutability: ["create"]' value.
+
+## Why the rule is important
+
+ A PATCH operation is used to update properties of a resource. So, If the resource has 'X' number of properties and if you wish to change one of them, then a PATCH request could be sent with a value for that specified property. In other words, all the properties in the PATCH request are updated. Now, if any of the values are marked as required/default, it would force the system to update it always which is not the intention of the PATCH operation.
```

Comment:
A PATCH operation is used to update properties of a resource.  Therefore, if the resource has 'X' number of properties and the purpose is to update one of them, then a PATCH request could be sent with a value for that specified property. Considering all properties in the PATCH request are to be updated, if any of values in PATCH request are marked as required/default, the updates are forced to happen although it may not be the intention of the PATCH operation.